htdocs/core/tpl/extrafields_list_search_input.tpl.php

```diff
@@ -20,12 +20,12 @@
 				$align=$extrafields->getAlignFlag($key);
 				$typeofextrafield=$extrafields->attributes[$extrafieldsobjectkey]['type'][$key];
 				print '<td class="liste_titre'.($align?' '.$align:'').'">';
-				if (in_array($typeofextrafield, array('varchar', 'int', 'double', 'select')) && empty($extrafields->attributes[$extrafieldsobjectkey]['computed'][$key]))
+				if (in_array($typeofextrafield, array('varchar', 'int', 'double')) && empty($extrafields->attributes[$extrafieldsobjectkey]['computed'][$key]))
 				{
 					$crit=$val;
 					$tmpkey=preg_replace('/search_options_/','',$key);
 					$searchclass='';
-					if (in_array($typeofextrafield, array('varchar', 'select'))) $searchclass='searchstring';
+					if (in_array($typeofextrafield, array('varchar'))) $searchclass='searchstring';
```

Comment:
This means we can't search anymore into an extrafields of type "select" ?

Reply:
Hi @eldy , i don't understand what you mean.
Until my PR, search on an extrafield of type "select" was an input text but the filter was made on key and not on value, and this makes no sense.
That's why i replaced that with a select list, which works well and is more logical for this kind of extrafield